From: Gabriel Valcazar <gabriel.valcazar@digi.com>
Date: Thu, 7 Feb 2019 13:15:43 +0100
Subject: [PATCH] bluetooth.service: add Digi customizations

These changes include:

* Having the bluetooth stack start automatically on boot.

Signed-off-by: Gabriel Valcazar <gabriel.valcazar@digi.com>
---
 src/bluetooth.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/bluetooth.service.in b/src/bluetooth.service.in
index beb98ce0c887..16abe9b08881 100644
--- a/src/bluetooth.service.in
+++ b/src/bluetooth.service.in
@@ -34,5 +34,5 @@ NoNewPrivileges=true
 RestrictRealtime=true
 
 [Install]
-WantedBy=bluetooth.target
+WantedBy=multi-user.target
 Alias=dbus-org.bluez.service
